config/README.md

```diff
@@ -10,10 +10,13 @@ To reduce efforts and duplication we decided to adapt the generation approach to
 
 ## Overview
 
-The `tools/peribolosbuilder.go` contains our custom generation logic for the peribolos configuration.
+We are using our [community-tools](https://github.com/open-feature/community-tooling/) to merge our configuration.
```

Comment:
I think I understand what is meant by this, but I think it would be helpful to explain briefly what's meant by "merge" in this context. At first I thought it was a typo and you meant to say "manage".